### PR TITLE
Add `<meta charset="UTF-8">` to transfer letters & render emojis with Twemoji

### PR DESCRIPTION
**Pull request by sampoder**
## Summary of the problem

I think this is what is preventing emojis from rending.

## Describe your changes

Sets the character set by adding `<meta charset="UTF-8">`

And then use Twemoji to render these wonderful emojis.

image

### CODE REVIEW

File: app/views/ach_transfers/transfer_confirmation_letter.pdf.erb [hunk 1]
```diff
@@ -131,11 +155,12 @@
...
+    <p
+      style="font-size: 12pt; font-weight: bold; color: #434343; position: fixed; bottom: 12px; width: 100%; white-space: nowrap;">
       By the students, for the students.
     </p>
     <p style="font-size: 10pt; color: #999999; position: fixed; bottom: 0px; width: 100%; white-space: nowrap;">
       Hack Club is a 501(c)(3) public charity. Our nonprofit EIN is 81-2908499.
     </p>
   </div>
-</body>
+</body>
```
Comment (garyhtou) [inline app/views/ach_transfers/transfer_confirmation_letter.pdf.erb]:
```suggestion
</body>

```

File: app/views/disbursements/transfer_confirmation_letter.pdf.erb [hunk 2]
```diff
@@ -122,11 +149,12 @@
...
+    <p
+      style="font-size: 12pt; font-weight: bold; color: #434343; position: fixed; bottom: 12px; width: 100%; white-space: nowrap;">
       By the students, for the students.
     </p>
     <p style="font-size: 10pt; color: #999999; position: fixed; bottom: 0px; width: 100%; white-space: nowrap;">
       Hack Club is a 501(c)(3) public charity. Our nonprofit EIN is 81-2908499.
     </p>
   </div>
-</body>
+</body>
```
Comment (garyhtou) [inline app/views/disbursements/transfer_confirmation_letter.pdf.erb]:
```suggestion
</body>

```

File: app/views/disbursements/transfer_confirmation_letter.pdf.erb [hunk 3]
```diff
@@ -9,45 +24,54 @@
       padding-right: 40pt;
       display: block;
     }
-    
+
+    * {
+      font-family: 'Phantom Sans', 'Noto Emoji', sans-serif;
+    }
```
Comment (garyhtou) [inline app/views/disbursements/transfer_confirmation_letter.pdf.erb]: is this still needed?

**Review by garyhtou — APPROVED** [review]
Please update the PR body and add an image!

File: app/views/disbursements/transfer_confirmation_letter.pdf.erb (hunk 3 above)
Comment (garyhtou) [reply app/views/disbursements/transfer_confirmation_letter.pdf.erb]: cc @sarthaktexas

Comment (sarthaktexas) [reply app/views/disbursements/transfer_confirmation_letter.pdf.erb]: oh oops no it's not! SORRY

**Pull request merged**